modules/smart-agent_systemd-timers/conf/01-timer-has-not-run-for.yaml

```diff
@@ -0,0 +1,15 @@
+module: systemd
+name: "timer has not run for"
+transformation: false
+signals:
+  A:
+    metric: gauge.active_state.active
+  B:
+    metric: gauge.active_state.activating
+  signal:
+    formula: "(A+B)"
+rules:
+  major:
+    threshold: 1
+    comparator: "<"
+    lasting_duration: 30m
```

Comment:
Seems ok for a first version but we should probably think for something more generic

Reply:
As discussed in direct message, it is not easy to make it generic, so it is disabled by default and the default alerting is set to 1 day instead of 30 minutes.